controllers/user.js

```diff
@@ -1,15 +1,36 @@
 const User = require('../models/user');
+const Exercise = require('../models/workoutlog');
+
 
 function index(req, res, next) {
-    User.find({}, function(err, user) {
-      res.render('user/index', { user,
-      user: req.user });
-    });
-  }
+ const user = req.user 
```

Comment:
Codacy has a fix for the issue: [Missing semicolon.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7828699)
```suggestion
 const user = req.user; 
```